drivers/hmis/app/models/hmis/search/assessment_search.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Search::AssessmentSearch
+  attr_accessor :input
+  def initialize(input)
+    self.input = input
+  end
+
+  def results(scope)
+    scope.
+      yield_self(&method(:with_roles))
+  end
```

Comment:
If there were multiple kinds of filters, would this look like below? Just want to confirm my understanding.
```
    scope.
      yield_self(&method(:with_roles)).
      yield_self(&method(:with_statuses))
```

Reply:
Yes, that's right